modules/release-notes/pages/index.adoc

```diff
@@ -124,6 +135,47 @@ vertex function can also be used with vertex input parameters in the query-body
 
 == Fixed issues
 
+=== Fixed and Improved in 3.9.3
+
+==== Crashes and Deadlocks
+* Fixed crash when query thread would keep running after `JobListener` is shutdown (CORE-1730).
+* Fixed GPE crash when using `INSERT` with an `UNKNOWN` accumulator (GLE-5038).
+* Fixed crash when running interpret query with dynamic value (GLE-5600).
+* Fixed JSON Viewer crash on browser when displaying a large response (TOOLS-2283).
+
+==== Functionality
+* Fixed issue of missing files in AdminPortal when downloading an output (TOOLS-2499).
+* Fixed issue primary id could not set a color config for a vertex (TOOLS-2478).
+* Fixed issue where administrator could privilege of vertex `primary_id` in Role Management page of Admin Portal (TOOLS-2430).
+* Fixed issue where error messages of abort requests created large file sizes (CORE-2705).
+* Fixed issue where GPE was still in warmup while running chaos testing (CORE-2670).
+* Fixed issue where user could download query results from the label “View Table Result” without proper privilege (TOOLS-2404).
+* Fixed issue where multiple vertex types return when selecting one vertex type (GLE-5325).
+* Fixed issue of show statement parser error for catalog object created using create statement (GLE-5719).
+* Fixed support for `outdegree()` with a collection of edge types in `INTERPRET` mode (GLE-5476).
+* Fixed issue where delete or edit vertex attributes were incorrect in GST (TOOLS-2258).
+* Fixed issue in GraphStudio when "=” appeared as a missing parameter in an `INTERPRET` query (TOOLS-2298).
+* Fixed argument error when passing a negative float parameter in GSQL cli in a `{key: value}` format. (GLE-6004).
+* Fixed GSE crash when `lease_expire_time_ms_` was not updating (CORE-2693).
+* Fixed issue where AWS credential user with access permission failed to `create data_source` to specified bucket (TP-4049).
+* Fixed issue of wrong result for `DISTINCT` with aggregation functions (GLE-3386).
+
+==== Improvements
+* Improved Insights table widget to be more consistent with graph studio (TOOLS-2375).
+* Improved insights search for two replicas when generating a pattern (TOOLS-2443).
+* Improved logs to show sso username and login related information (GLE-5970).
+* Improved precision of `INT` and `UNIT` (CORE-2703).
+* Improved error message details when during a failure to find a request record when deleting (CORE-2686).
+* Improved API user existent check for when permission is not needed (GLE-5678).
+* Improved performance of edit vertex when vertex 200 attributes. (TOOLS-2107)
+* Added memory check in `DistrbutedTopologyStore` (CORE-2496).
+* LDAP now has more detailed error messages (GLE-5559).
+* Implemented connector fail status to show error messages in GSQL console (TP-3465).
+* Added link to TigerGraph Kubernetes preview documentation on GitHub (TP-4039).
+* Expended support for `gcollect` (TP-4127).
```

Comment:
Typo.  Also, gcollect is not yet documented...
See DOC-1807.

Reply:
removed the gcollect items